OnlineShop.Services.Catalog.Infrastructure/Repositories/Implementations/ProductRepository.cs

```diff
@@ -0,0 +1,80 @@
+﻿using MongoDB.Driver;
+using OnlineShop.Services.Catalog.Application.Exceptions;
+using OnlineShop.Services.Catalog.Domain.Models.Data;
+using OnlineShop.Services.Catalog.Domain.Repositories.Interfaces;
+using OnlineShop.Services.Catalog.Infrastructure.Data.Interfaces;
+using System.Linq.Expressions;
+
+namespace OnlineShop.Services.Catalog.Infrastructure.Repositories.Implementations
+{
+    public class ProductRepository : IProductRepository
+    {
+        private readonly ICatalogContext _catalogContext;
+
+        public ProductRepository(ICatalogContext catalogContext)
+        {
+            _catalogContext = catalogContext;
+        }
+
+        public async Task<IEnumerable<Product>> GetAllAsync()
+            => await _catalogContext
+                     .Products
+                     .Find(p => true)
+                     .ToListAsync();
```

Comment:
To maintain a more conventional coding style, it's recommended to steer clear of employing expression-bodied methods

Reply:
so, not to use expression-bodied methods at all, even in cases where method contains just one expression?

Reply:
Yes, it's important to adhere to a unified coding style